src/Features/Protocol/LanguageServices/Lsp/DocumentSymbol.cs

```diff
@@ -0,0 +1,62 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+
+using System.Collections.Generic;
+using System.Runtime.Serialization;
+
+// Straight copy until new Microsoft.VisualStudio.LanguageServer.Protocol nuget is published with this type in
+namespace Microsoft.VisualStudio.LanguageServer.Protocol
+{
+    /// <summary>
+    /// Represents programming constructs like variables, classes, interfaces etc. that appear in a document. Document symbols can be
+    /// hierarchical and they have two ranges: one that encloses its definition and one that points to its most interesting range,
+    /// e.g. the range of an identifier.
+    /// </summary>
+    [DataContract]
```

Comment:
now your PR in, if we have Nuget with your changes, I think we should publish that nuget and use that and remove this.

Reply:
Will do that once Tina's changes for folding range are in as well (probably in a followup PR)

Reply:
https://github.com/dotnet/roslyn/projects/45#card-20051462